### ISSUE TITLE
Team update - Feb 01, 2023

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#193]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by steff456** [comment]
**Updates from last week :heavy_check_mark:**
- I did some maintenance stuff in `accessibility-pygments`
   - I added Tania as maintainer to PyPI and conda-forge 
- I continue doing the collapse menu work
   - Darian already approved the PR - he just found two bugs, one that's not related to my PR and other one that is. So I'm working right now in finding that bug
   - I already added an optional flag for downstream libraries to decide if they want to use the collapse menu or not. 

**Challenges I faced and things I'd like assistance with 🙏**
- I'm trying to find the source of the bug 😅

**My availability for next week**
- I'll be here

**Important items for discussion 💬**
- I don't have anything for today (:

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌**

**Updates from last week :heavy_check_mark:**
- #192 Updated test notebook and opened [PR to add notebook to the testing repo](https://github.com/Quansight-Labs/JupyterLab-user-testing/pull/1). I imagine this needs to go somewhere more specific or to a different branch, but I figured it's a start. 
- #184 draft is still not done. 😦 End-of-week remains my goal timeline.

**Challenges I faced and things I'd like assistance with 🙏**
- I don't think I have anything now.

**My availability for next week**
- I'll be here all week. I've been a bit occupied with another project, though, and that may continue needing some extra focus.

**Important items for discussion 💬**
- I don't think I have anything right now.

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- Thank you all for your patience with me as I get better.

**Updates from last week :heavy_check_mark:**
- Staying alive. Nothing new for me since our last team update. I've been heads down on the Quansight website and Lab annual report.

**My availability for next week**
- Here

**Important items for discussion 💬**
- How much effort should go into accessibility fixes in future releases of 3 versus 4? Context: [JupyterLab 3.6.0rc1 - style changes #13870](https://github.com/jupyterlab/jupyterlab/issues/13870).

**Comment by trallard** [comment]
@gabalafou 
	- He is still alive after being super sick 😷 
	- Planning to work on the `tab trap` doc for the accessibility meeting
	- when do JLab sunset or stop maintaining previous versions?
	- Focus on a11y work on `master` (which is 4.x now)

@isabela-pf 
	- @trallard to review https://github.com/Quansight-Labs/JupyterLab-user-testing/pull/1
	- Places where we cannot pay people: India, Estonia, and most of LatAm so we can start recruiting testers

@steff456 
	- maintenance in our pygments repo
	- 🍔  menu is moving forward -> tests + bug squashing needed
	- next: pygments fixes 
	- need to improve the highlight colours further so that they are WCAG conformant https://www.w3.org/TR/WCAG22/#contrast-minimum

**Comment by isabela-pf** [comment]
Related to question of focus for accessibility fixes and versions.
* [JupyterLab releases](https://github.com/jupyterlab/jupyterlab/releases)
* [jupyterlab/jupyterlab #12397 Add version support timelines to documentation](https://github.com/jupyterlab/jupyterlab/issues/12397)
